Tiltfile

```diff
@@ -27,16 +27,15 @@ deku_yaml = make_deku_yaml(no_of_deku_nodes)
 # Run docker-compose
 docker_compose(["./docker-compose.yml", deku_yaml])
 
-dc_resource("db", labels=["database"], trigger_mode=TRIGGER_MODE_MANUAL, auto_init=False)
-dc_resource("elastic", labels=["database"], trigger_mode=TRIGGER_MODE_MANUAL, auto_init=False)
-
+dc_resource("db", labels=["tezos"])
```

Comment:
These changes makes `tilt ci` needlessly slow and noisy and there's not a good way to disable them. How important is it to always run these resources?
@d4hines